notebooks/01.02-PyRosetta-Google-Drive-Usage-Example.ipynb

```diff
@@ -42,11 +42,12 @@
    "outputs": [],
    "source": [
     "# Notebook setup\n",
-    "\n",
-    "!pip install pyrosettacolabsetup\n",
-    "import pyrosettacolabsetup\n",
-    "pyrosettacolabsetup.setup()\n",
-    "print (\"Notebook is set for PyRosetta use in Colab.  Have fun!\")"
+    "import sys\n",
+    "if 'google.colab' in sys.modules:\n",
+    "    !pip install pyrosettacolabsetup\n",
+    "    import pyrosettacolabsetup\n",
+    "    pyrosettacolabsetup.setup()\n",
+    "    print (\"Notebook is set for PyRosetta use in Colab.  Have fun!\")"
```

Comment:
Why not print this in the setup function?

Reply:
Agreed! That would be cool. Here I just re-implemented the setup functionality that was already implemented in other notebooks, just wrapped in an `if 'google.colab' in sys.modules:`.